x-pack/plugins/reporting/server/routes/lib/jobs_query.ts

```diff
@@ -5,161 +5,162 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { errors as elasticsearchErrors } from 'elasticsearch';
+import {
+  CountParams,
+  CountResponse,
+  errors as elasticsearchErrors,
+  SearchParams,
+  SearchResponse,
+} from 'elasticsearch';
 import { get } from 'lodash';
 import { ReportingCore } from '../../';
-import { AuthenticatedUser } from '../../../../security/server';
-import { JobSource } from '../../types';
+import { JobSource, ReportingUser } from '../../types';
 
-const esErrors = elasticsearchErrors as Record<string, any>;
-const defaultSize = 10;
-
-interface QueryBody {
-  size?: number;
-  from?: number;
-  _source?: {
-    excludes: string[];
-  };
-  query: {
-    constant_score: {
-      filter: {
-        bool: {
-          must: Array<Record<string, any>>;
-        };
-      };
-    };
-  };
+// FIXME: typescript definition does not have the proper search API field name!
+declare module 'elasticsearch' {
```

Comment:
The Elasticsearch API assured me that it is supposed to be `_source_excludes` not `_sourceExclude`